docs/changes/utils/13004.update.rst

```diff
@@ -0,0 +1 @@
+Re-added maia.usno.navy.mil to the list of IERS A data download sources.
```

Comment:
`update` is not a valid category. Please see https://github.com/astropy/astropy/blob/main/docs/changes/README.rst